modules/monitoring-configuring-different-alert-receivers-for-default-platform-alerts-and-user-defined-alerts.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-different-alert-receivers-for-default-platform-alerts-and-user-defined-alerts_{context}"]
+= Configuring different alert receivers for default platform alerts and user-defined alerts
+
+You can configure different alert receivers for default platform alerts and user-defined alerts to ensure the following results:
+
+* All default platform alerts are sent to a receiver owned by the team in charge of these alerts.
+* All user-defined alerts are sent to another receiver so that the team can focus only on platform alerts.
+
+You can achieve this by using the `openshift_io_alert_source="platform"` label that is added by the Cluster Monitoring Operator to all platform alerts.
+
+.Prerequisites
+
+ifndef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `dedicated-admin` role.
+endif::openshift-dedicated,openshift-rosa[]
+
+.Procedure
```

Comment:
I wonder if we should go down to this level of detail (real-life Alertmanager configurations are usually more complex than this example). Maybe we should only document the existence and meaning of the `openshift_io_alert_source` label + the 2 matcher examples?


```
Platform alerts have the `openshift_io_alert_source="platform"` label
while user-defined alerts have no `openshift_io_alert_source` label.

You can use route matchers in the Alertmanager configuration to match alerts to the correct destination:
* Use the `openshift_io_alert_source="platform"` matcher to match platform alerts.
* Use the `openshift_io_alert_source!="platform"` matcher to match user-defined alerts.
````

Reply:
Hello @simonpasquier I understand what you mean here, I was wondering too how detailed this should be. I was also thinking about the possibility to include it here: https://docs.openshift.com/container-platform/4.15/monitoring/managing-alerts.html#configuring-alert-receivers_managing-alerts to the step 5.

But my concert here was the following:
It was mentioned in the doc issue that this is very common configuration. For this reason I believe having this as a separate section is better than including it in the already existing  section.
 
The second solution I was also thinking about was making this more of a conceptual chapter, instead of giving these precise steps.  In that case I would keep the information from the introduction and add something similar to what you suggested above.

Or the third option would be to keep it as it is right now.

WDYT? Maybe the second option would be the best after all?

Reply:
@eliska - I think the second option is the best. It keeps things at a high conceptual level rather than providing an overly simplistic example of a specific implementation. I think that, as @simonpasquier points out, it's enough just to provide info about what the labels are and what they do. Implementation teams should be knowledgeable enough to know how to apply the labels to their config.

Reply:
Hello @bburt-rh thank you very much for the confirmation!